src/App.js

```diff
@@ -1,20 +1,74 @@
-import './App.css';
+// import './App.css';
```

Comment:
- [ ] Kindly, remove this all piece of commented code in order to improve your repo's cleanliness. 👍